proctor-consumer/src/main/java/com/indeed/proctor/consumer/AbstractGroupsManager.java

```diff
@@ -83,26 +96,56 @@ protected ProctorResult determineBucketsInternal(final Identifiers identifiers,
                     emptyMap()
             );
         }
-        final ProctorResult proctorResult = proctor.determineTestGroups(identifiers, context, forcedGroups);
+        final ProctorResult proctorResult = proctor.determineTestGroups(
+                identifiers,
+                context,
+                forceGroupsOptions,
+                emptyList()
+        );
```

Comment:
What's the point of this change? It looks `proctor.determineTestGroups(identifiers, context, forcedGroups)` internally calls `determineTestGroups(identifiers, inputContext, forceGroups, Collections.emptyList())`. So the behavior is identical.

Reply:
Please note that `forcedGroups` is changed to `forceGroupsOptions` (`ForceGroupsOptions` class) here. I didn't add a method `determineTestGroups` with `(identifiers, context, forceGroupsOptions)` signature to prevent too many methods in the class so `Collections.emptyList()` was necessary.

Reply:
Got it. Can't we mark all the `determineTestGroups` with `Map forceGroups` as deprecated?

Reply:
It seems all of them except `(identifiers, context, Map)` has deprecated tag. I will add it to the method too.

Reply:
done https://github.com/indeedeng/proctor/pull/64/commits/bea8fd88824234e593aed780a4fba9d6206d0834